api/src/main/java/org/openmrs/module/ugandaemrreports/library/Moh105CohortLibrary.java

```diff
@@ -211,4 +211,45 @@ public CohortDefinition hasObsAndEncounter(String encounterType, Concept q, Conc
         cd.setCompositionString("ancEncounter AND hasObs");
         return cd;
     }
+
+	/**
+	 * Number Tetanus Immunizations done
+	 * 
+	 * @return CohortIndicator
+	 */
+	public CohortDefinition tetanusImmunizationsDone(int doseNumber) {
```

Comment:
@ssmusoke.This has been done. Also added the link to the report in the ugandaemr_reports_app.json file